tests/test_MigrateData.py

```diff
@@ -0,0 +1,1072 @@
+import pytest
+
+import KoalaBot
+from utils.KoalaDBManager import KoalaDBManager
+from utils.MigrateData import MigrateData
+
+database_manager = KoalaDBManager(KoalaBot.DATABASE_PATH, KoalaBot.DB_KEY, KoalaBot.CONFIG_DIR)
+database_manager.create_base_tables()
+migrate_database = MigrateData(database_manager)
+
+TABLE_NAME = "{TABLE_NAME}"
+DROP_TABLE_SQL = f"DROP TABLE {TABLE_NAME}"
```

Comment:
are these used?

Reply:
so i could use a for loop to drop tables so i could start again more easily, if im squashing the create and populate methods then I'll get rid of this as well